codec_iana.md

```diff
@@ -48,14 +48,15 @@ A_AAC/MPEG4/LC/SBR | 2 | Low Complexity with Spectral Band Replication | This do
 A_AAC/MPEG4/LTP | 2 | Long Term Prediction | This document, (#a-aac-mpeg4-ltp)
 A_AAC/MPEG4/MAIN | 2 | MPEG4 Main Profile | This document, (#a-aac-mpeg4-main)
 A_AAC/MPEG4/SSR | 2 | Scalable Sampling Rate | This document, (#a-aac-mpeg4-ssr)
-A_AC3 | 2 | Dolby Digital AC-3 | This document, (#a-ac3)
-A_AC3/BSID9 | 2 | Dolby Digital AC-3 | This document, (#a-ac3-bsid9)
-A_AC3/BSID10 | 2 | Dolby Digital AC-3 | This document, (#a-ac3-bsid10)
+A_AC3 | 2 | Dolby Digital / AC-3 | This document, (#a-ac3)
+A_AC3/BSID9 | 2 | Dolby Digital / AC-3 | This document, (#a-ac3-bsid9)
+A_AC3/BSID10 | 2 | Dolby Digital / AC-3 | This document, (#a-ac3-bsid10)
 A_ALAC | 2 | ALAC (Apple Lossless Audio Codec) | This document, (#a-alac)
 A_ATRAC/AT1 | 2 | Sony ATRAC1 Codec | This document, (#a-atrac-at1)
 A_DTS | 2 | Digital Theatre System | This document, (#a-dts)
 A_DTS/EXPRESS | 2 | Digital Theatre System Express | This document, (#a-dts-express)
 A_DTS/LOSSLESS | 2 | Digital Theatre System Lossless | This document, (#a-dts-lossless)
+A_EAC3 | 2 | Dolby Digital Plus / Enhanced AC-3 | This document, (#a-eac3)

```

Comment:
```suggestion
A_EAC3 | 2 | Dolby Digital Plus / E-AC-3 | This document, (#a-eac3)
```
For coherency, use `Dolby Digital / AC-3` and `Dolby Digital Plus / E-AC-3` everywhere.
(also in codec name for AC-3)